src/vivarium_gates_mncnh/validation/formatting.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+from vivarium_testing_utils.automated_validation.constants import DRAW_INDEX, SEED_INDEX
+from vivarium_testing_utils.automated_validation.data_transformation import calculations
+from vivarium_testing_utils.automated_validation.data_transformation.formatting import (
+    SimDataFormatter,
+)
+
+
+class TotalLiveBirths(SimDataFormatter):
+    """Formatter for simulation data that contains total live births."""
+
+    def __init__(self) -> None:
+        self.measure = "live_births"
+        self.entity = "total"
+        # Hardcoding raw dataset name due to custom observers
+        self.raw_dataset_name = "births"
+        self.unused_columns = [
+            "measure",
+            "entity_type",
+            "entity",
+            "sub_entity",
+        ]
+        self.filter_value = "total"
+        # NOTE: Sub entity column is currently null but will have useful values at some point
+        self.filters = {"sub_entity": [self.filter_value]}
+        self.name = f"{self.filter_value}_{self.measure}"
+
+    def format_dataset(self, dataset: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Something I was thinking about here is the births observer output has a lot of stratifications that seem unnecessary and will not be in all of the outputs that would be used for the numerator data. Maybe we just want to get rid of all columns except "child_sex" so we only have to cast across levels from the numerator. I do not think we need to change anything now.